iphone/Classes/TiUIWebView.m

```diff
@@ -177,6 +179,9 @@ -(UIWebView*)webview
 			[spinner sizeToFit];
 			[spinner startAnimating];
 		}
+        if (defaultUserAgent == nil) {
+            defaultUserAgent = [webview stringByEvaluatingJavaScriptFromString:@"navigator.userAgent"];
```

Comment:
What happens on the second call, when there is no default user-agent anymore? Then it could never be resetted again?